### PR TITLE
Implement Nylium Blocks [closed because of conflicts]

### PR DESCRIPTION
**Pull request by pandaaaBE**
## Introduction
Implement both Nether Nylium Blocks

closed this by deleting the head repository

### API changes
`VanillaBlocks::CRIMSON_NYLIUM(), VanillaBlocks::WARPED_NYLIUM()`

image

### CODE REVIEW

**Comment by pandaaaBE** [comment]
Nylium should drop,
with Slik Touch: the same block
with pickaxe: Netherrack
with hand or other tools: nothing

I have the bug that it just drops sometimes with pickaxe but always with slik touch, somebody know why?

File: src/block/NetherNylium.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	public function ticksRandomly() : bool{
+		return true;
+	}
+
+	public function onRandomTick() : void{
+		$world = $this->position->getWorld();
+		$lightAbove = $world->getFullLightAt($this->position->getFloorX(), $this->position->getFloorY() + 1, $this->position->getFloorZ());
+		if($lightAbove < 4 && $world->getBlockAt($this->position->getFloorX(), $this->position->getFloorY() + 1, $this->position->getFloorZ())->getLightFilter() >= 2){
+			//nylium dies
+			BlockEventHelper::spread($this, VanillaBlocks::NETHERRACK(), $this);
```
Comment (ShockedPlot7560) [inline src/block/NetherNylium.php:48]: This is the wrong event maybe ?

Comment (pandaaaBE) [reply src/block/NetherNylium.php:48]: This is the code for grass, and nylium (to netheerack) have the same effect in vanilla..

**Comment by pandaaaBE** [comment]
PR should now be finished

**Comment by pandaaaBE** [comment]
still waiting for final review

File: src/block/NetherNylium.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		return [
+			VanillaBlocks::NETHERRACK()->asItem()
+		];
+	}
+
+	public function ticksRandomly() : bool{
+		return true;
+	}
+
+	public function onRandomTick() : void{
```
Comment (ShockedPlot7560) [inline src/block/NetherNylium.php:45]: Nylium decay when covered with solid block or snow.

**Comment by dktapps** [comment]
Looks like bone meal logic is missing.

**Pull request closed without merging**